core/src/main/scala/io/finch/EndpointModule.scala

```diff
@@ -289,6 +290,22 @@ trait EndpointModule[F[_]] {
   def asyncBody(implicit F: Effect[F]): Endpoint[F, AsyncStream[Buf]] =
     Endpoint.asyncBody[F]
 
+  /**
+    * An alias for [[Endpoint.streamBody]]
+    */
+  def streamBody[S[_[_],_], A, CT <: String](implicit
```

Comment:
Another question about this. I'm trying to figure out if we can simply the type signature given we have the `io.finch.fs2` import in place? I'm thinking along the lines of `EndpointModule` but for streaming types. Don't have anything concrete in mind, just wanted to pointed out I'm thinking if there anything we can do.

Reply:
Sure
https://github.com/finagle/finch/pull/1042/commits/9e86108b117e517b2193989199bf33d3a3712449